beta/google-cloud-run/README.md

```diff
@@ -0,0 +1,187 @@
+
+```sh
+curl --silent --show-error \
+  https://raw.githubusercontent.com/1Password/scim-examples/solutions/pike/google-cloud-run/beta/google-cloud-run/op-scim-bridge.yaml |
+  gcloud run services replace - &&
+  gcloud run services add-iam-policy-binding op-scim-bridge --member=allUsers --role=roles/run.invoker &&
+  gcloud run services describe op-scim-bridge --format="value(status.url)"
+```
+
+<!-- !TODO: Replace the URL above with the link to its path when it is merged into main.
+```sh
+curl --silent --show-error \
+  https://raw.githubusercontent.com/1Password/scim-examples/main/beta/google-cloud-run/op-scim-bridge.yaml |
+  gcloud run services replace - &&
+  gcloud run services add-iam-policy-binding op-scim-bridge --member=allUsers --role=roles/run.invoker
+```
+-->
+
+The final line of the above chained command should output a URL for the HTTPS endpoint provded by Cloud Run. This is your **SCIM Bridge URL**.
+
+## Step 4: Test your SCIM bridge
+
+Use your SCIM Bridge URL to test the connection and view status information. For example:
+
+```sh
+curl --silent --show-error --request GET --header "Accept: application/json" \
+  --header "Authorization: Bearer mF_9.B5f-4.1JqM" \
+  https://op-scim-bridge-example-uc.a.run.app/health
+```
+
+Replace `mF_9.B5f-4.1JqM` with your bearer token and `https://op-scim-bridge-example-uc.a.run.app` with your SCIM Bridge URL.
+
+<details>
+<summary>Example JSON response:</summary>
+
+```json
+{
+  "build": "209031",
+  "version": "2.9.3",
+  "reports": [
+    {
+      "source": "ConfirmationWatcher",
+      "time": "2024-04-25T14:06:09Z",
+      "expires": "2024-04-25T14:16:09Z",
+      "state": "healthy"
+    },
+    {
+      "source": "RedisCache",
+      "time": "2024-04-25T14:06:09Z",
+      "expires": "2024-04-25T14:16:09Z",
+      "state": "healthy"
+    },
+    {
+      "source": "SCIMServer",
+      "time": "2024-04-25T14:06:56Z",
+      "expires": "2024-04-25T14:16:56Z",
+      "state": "healthy"
+    },
+    {
+      "source": "StartProvisionWatcher",
+      "time": "2024-04-25T14:06:09Z",
+      "expires": "2024-04-25T14:16:09Z",
+      "state": "healthy"
+    }
+  ],
+  "retrievedAt": "2024-04-25T14:06:56Z"
+}
+```
+
+</details>
+<br />
+
+Similar information is presented graphically by accessing your SCIM Bridge URL in a web browser. Sign in with your bearer token to view status information and download container log files.
+
+## Step 5: Connect your identity provider
+
+> [!IMPORTANT]
+> Additional steps are required to [connect 1Password SCIM Bridge to Google Workspace](./google-workspace/README.md).
+
+To finish setting up automated user provisioning, [connect your identity provider to your SCIM Bridge](https://support.1password.com/scim/#step-3-connect-your-identity-provider).
+
```

Comment:
Add update steps for CLI: The suggestion box does not handle the code block section formatting correctly so we will need to fix these through a commit. 

## Update your SCIM bridge

> [!TIP]
> Check for 1Password SCIM Bridge updates on the [SCIM bridge releases notes website](https://releases.1password.com/provisioning/scim-bridge/).

 > [!NOTE]
 > If you used Google Workspace, follow the [steps here to update your deployment](./google-workspace/README.md#Update-your-SCIM-bridge-when-Google-Workspace-is-your-IdP)

1. Connect to your Cloud Shell and run the following command:

```sh
curl --silent --show-error \
  https://raw.githubusercontent.com/1Password/scim-examples/main/beta/google-cloud-run/op-scim-bridge.yaml |
  gcloud run services replace -
```

2. Enter your SCIM bridge URL in a browser and sign in with your bearer token.
3. Check the top left-hand side of the page to verify you're running the updated version of the SCIM Bridge.

After you sign in to your SCIM bridge, the [Automated User Provisioning page](https://start.1password.com/integrations/active/) in your 1Password account will also update with the latest access time and SCIM bridge version.

Reply:
Apparently the solution is 4 backticks instead of 3 for the suggestion block (see https://github.com/github/markup/issues/1687#issuecomment-1836916420):

````suggestion

## Update your SCIM Bridge

> [!TIP]
> Check for 1Password SCIM Bridge updates on the [SCIM bridge releases notes website](https://releases.1password.com/provisioning/scim-bridge/).

> [!IMPORTANT]
> If you used Google Workspace, follow the [steps here to update your deployment](./google-workspace/README.md#Update-your-SCIM-bridge-when-Google-Workspace-is-your-IdP)

1. Connect to your Cloud Shell and run the following command:

```sh
curl --silent --show-error \
  https://raw.githubusercontent.com/1Password/scim-examples/main/beta/google-cloud-run/op-scim-bridge.yaml |
  gcloud run services replace -
```

2. Enter your SCIM bridge URL in a browser and sign in with your bearer token.
3. Check the top left-hand side of the page to verify you're running the updated version of the SCIM Bridge.

After you sign in to your SCIM Bridge, the [Automated User Provisioning page](https://start.1password.com/integrations/active/) in your 1Password account will also update with the latest access time and SCIM Bridge version.

````